### DISCUSSION TITLE
Stage Channels

### DISCUSSION THREAD

**Discussion by KamikariCat** [discordjs/discord.js#5462] [Q&A] [answered]
Hey what about stage channels support?

**Comment by kyranet** [comment, marked as answer]
Stage channels will be supported starting with #5456.

**Reply by kyranet** [reply]
Yes, for all intents and purposes, stage channels work similarly to voice channels except two differences:

- Stage channels do not support videos (nor streams)
- Stage channels have a `REQUEST_TO_SPEAK` permission and action.

**Reply by kyranet** [reply]
It'll be available in `master` branch soon, as for a release, there's no ETA.
